internal/cuckoo/filter.go

```diff
@@ -0,0 +1,169 @@
+package cuckoo
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/rand"
+)
+
+const (
+	BucketSize           = 4
+	NumBuckets           = 524288
```

Comment:
why number of buckets is fixed? shouldn't it be dependent on number of store values?

Reply:
It's a fixed parameter actually - the number of buckets should be consistent across all nodes.